source/arm9/sound.c

```diff
@@ -122,6 +138,8 @@ MicCallback micCallback = 0;
 void micBufferHandler(int bytes, void* user_data){
 	FifoMessage msg;
 
+	// TODO: Do we need a lock here?
```

Comment:
This is inside an IRQ! Don't use yielding here.

Reply:
Ok, I'll change the note.